Projects/UOContent/Items/Construction/Chairs/Chairs.cs

```diff
@@ -171,14 +171,14 @@ public override void Deserialize(IGenericReader reader)
     }
 
     [DynamicFliping, Flippable(0x2DEB, 0x2DEC, 0x2DED, 0x2DEE)]
-    public class BigElvenChair : Item
+    public class CozyElvenChair : Item
```

Comment:
Let's add a TypeAlias for BigElvenChair so we don't break people's saves.

Reply:
Reverted back as name change was unneeded! Still shows proper name.